packages/fiori/src/SideNavigation.ts

```diff
@@ -155,21 +101,109 @@ const metadata = {
  * @appenddocs sap.ui.webc.fiori.SideNavigationItem sap.ui.webc.fiori.SideNavigationSubItem
  * @public
  */
+
+@customElement("ui5-side-navigation")
+@fastNavigation
+/**
+ * Fired when the selection has changed via user interaction
+ *
+ * @event sap.ui.webc.fiori.SideNavigation#selection-change
+ * @param {HTMLElement} item the clicked item.
+ * @allowPreventDefault
+ * @public
+ */
+@event("selection-change", {
+	detail: {
+		item: { type: HTMLElement },
+	},
+})
 class SideNavigation extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines whether the <code>ui5-side-navigation</code> is expanded or collapsed.
+	 *
+	 * @public
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @name sap.ui.webc.fiori.SideNavigation.prototype.collapsed
+	 */
+	@property({ type: Boolean })
+	collapsed!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Object })
+	_popoverContent?: {
```

Comment:
If you find good name for this, a type could be nice for this object to be explicitly named.